code/modules/mob/new_player/new_player.dm

```diff
@@ -103,7 +106,7 @@
 
 	output += "</center>"
 
-	var/datum/browser/popup = new(src, "playersetup", "<div align='center'>New Player Options</div>", 220, 290)
+	var/datum/browser/popup = new(src, "playersetup", "<div align='center'>New Player Options</div>", 240, 330)
```

Comment:
Does this finally fix the weirdness where youre in lobby midround and it doesnt show properly.
![image](https://user-images.githubusercontent.com/25063394/94363053-6a750080-00b7-11eb-8df6-17170afe9165.png)

Reply:
Assuming "the weirdness" you're talking about is the fact it has a scroll bar, yes.